python/social_bot/envs/grocery_ground.py

```diff
@@ -86,6 +86,8 @@ def __init__(self,
             random_range (float): the goal's random position range
             random_goal (bool): if ture, teacher will randomly select goal from the object list each episode
         """
+        if goal_name is None:
```

Comment:
it seems this can be removed

Reply:
Good observation Jiangtao.  I put it here just in case the user passes in `goal_name = None` in the gin file, in which case, a default value is still provided here.

Reply:
Maybe it's better to alert the user if that happens?

Reply:
It's ok just remove it. No one would set goal_name=None in gin file...